silence/silence.go

```diff
@@ -447,33 +501,14 @@ func QTimeRange(start, end time.Time) QueryParam {
 }
 
 // QMatches returns silences that match the given label set.
-func QMatches(set map[string]string) QueryParam {
+func QMatches(set model.LabelSet) QueryParam {
```

Comment:
I never was much of a fan of these custom types. That's why i deliberately left it out of this API. We would drag in a package dependency without ever using anything from that package.